dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -218,4 +222,185 @@ public Response deleteEnvironmentPushHistory(@Context HttpServletRequest request
         return responseResource.response( "true" );
 	}
 
+    /**
+     * Deletes all bundles by identifier
+     * @param request
+     * @param response
+     * @param deleteBundlesByIdentifierForm
+     * @return
+     */
+	@DELETE
+    @Path("/ids")
+    @Produces("application/json")
+    public Response deleteBundlesByIdentifiers(@Context final HttpServletRequest request,
+                                               @Context final HttpServletResponse response,
+                                               final DeleteBundlesByIdentifierForm  deleteBundlesByIdentifierForm) {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requiredBackendUser(true)
+                .requiredFrontendUser(false)
+                .requestAndResponse(request, response)
+                .rejectWhenNoUser(true)
+                .init();
+
+        Logger.info(this, "Deleting the bundles: " + deleteBundlesByIdentifierForm.getIdentifiers()
+                + " by the user: " + initData.getUser().getUserId());
+
+        try {
+
+            for (final String bundleId : deleteBundlesByIdentifierForm.getIdentifiers()) {
+
+                this.bundleAPI.deleteBundleAndDependencies(bundleId, initData.getUser());
+            }
+
+            return Response.ok(new ResponseEntityView("All bundles deleted")).build();
+        } catch (DotDataException e) {
+
+            Logger.error(this.getClass(),
+                    "Exception on deleteBundlesByIdentifiers, couldn't delete the identifiers: " + deleteBundlesByIdentifierForm.getIdentifiers() +
+                            ", exception message: " + e.getMessage(), e);
+            return ResponseUtil.mapExceptionResponse(e);
+        }
+    } // deleteBundlesByIdentifiers.
+
+    /**
+     * Deletes bundles older than a date. (unsent are not going to be deleted)
+     * @param request
+     * @param response
+     * @param olderThan
+     * @return
+     */
+    @DELETE
+    @Path("/olderthan/{olderThan}")
+    @Produces("application/json")
+    public Response deleteBundlesOlderThan(@Context final HttpServletRequest request,
+                                           @Context final HttpServletResponse response,
+                                           @PathParam("olderThan") final DateParam olderThan) {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requiredBackendUser(true)
+                .requiredFrontendUser(false)
+                .requestAndResponse(request, response)
+                .rejectWhenNoUser(true)
+                .init();
+
+        Logger.info(this, "Deleting the bundles older than: " + olderThan
+                + " by the user: " + initData.getUser().getUserId());
+
+        try {
+
+            return Response.ok(new ResponseEntityView(CollectionsUtils.map("bundlesDeleted",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "bundlesDeleted" appears 4 times in this file; the first occurrence is on line 292](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)